src/constants.ts

```diff
@@ -42,3 +44,4 @@ export const dataEntryClose = document.querySelector(
 export const dataEntryModal = document.querySelector(
```

Comment:
why are query selectors added here? wont that be run in pages where they are not needed? maybe the selector string here but do the actual selection in the page where it is needed

Reply:
changed them and now given wherever necessary only